src/com/animation.js

```diff
@@ -0,0 +1,204 @@
+/**
+ * Animation component
+ * 
+ * @author Deux Huit Huit
+ * 
+ * @uses bodymovin
+ * @uses jQuery
+ * @uses underscore.js
+ * 
+ * REQUIRE: https://cdnjs.cloudflare.com/ajax/libs/bodymovin/4.13.0/bodymovin.min.js
+ * DOCS: https://github.com/chenqingspring/bodymovin
+ */
+(function ($) {
+
+	'use strict';
+
+	App.components.exports('animation', function (options) {
+
+		var animation = $();
+		var renderer = null;
+		var isPlaying = false;
+
+		var defaults = {
+			renderer: 'svg',
+			loop: false
+		};
+
+		/**
+		 * Modify the speed of the animation
+		 * @param {Number} speed The speed of the animation
+		 */
+		var setSpeed = function (speed) {
+			renderer.setSpeed(speed || 1); // 1 is normal speed.
+		};
+
+		/**
+		 * Go to a specific moment in the animation and pause.
+		 * @param {Number} value The wanted position in the timeline
+		 * @param {Boolean} isFrame if the value is a specific time on the 
+		 * 							timeline or a specific frame
```

Comment:
???

Reply:
?

Reply:
indentation is wack. no tabs after chars also.
```suggestion
		 *  timeline or a specific frame
```